packages/flutter/test/widgets/image_filter_test.dart

```diff
@@ -116,11 +116,49 @@ void main() {
     );
     await expectLater(
       find.byType(ImageFiltered),
-      matchesGoldenFile('image_filter_matrix.png'),
+      matchesGoldenFile('image_filter_matrix_m2.png'),
     );
   });
 
-  testWidgets('Image filter - matrix with offset', (WidgetTester tester) async {
+  testWidgets('Image filter - matrix', (WidgetTester tester) async {
+    final ImageFilter matrix = ImageFilter.matrix(Float64List.fromList(<double>[
+      0.5, 0.0, 0.0, 0.0, //
```

Comment:
I'm actually not very familiar with ColorFilter and ImageFiler, why do we end each line with a "//"?